.tugboat/steps/3-build.sh

```diff
@@ -6,5 +6,6 @@
 set -eux
 echo "Building..."
 
-./vendor/bin/task build
-./vendor/bin/task update
+./vendor/bin/task tugboat:build
+
+./vendor/bin/task tugboat:update
```

Comment:
We have concerns that these would break builds for existing projects that do not rename their old commands if they didn't replace them with these new task commands `<service>:<action>` cc @deviantintegral